hack/gen-swagger-doc/gen-swagger-docs.sh

```diff
@@ -22,8 +22,9 @@ set -o pipefail
 
 cd /build/
 
-wget "$2" -O input.json
-wget "$3" -O register.go
+wget "$2" -O register.go
+
+cp /swagger-source/"$1".json input.json
```

Comment:
Add a comment saying we do this since gendocs requires the file name to be input.json